tests/e2e/project.spec.ts

```diff
@@ -361,72 +359,73 @@ test("only one modal can be open at a time", async ({ page }) => {
   await expect(page.locator("text=bob hyyzz9")).not.toBeVisible();
   await expect(page.locator("text=feature/branch")).not.toBeVisible();
 });
-
-test.describe("browser error handling", () => {
-  test("error appears when folder can't be loaded", async ({ page }) => {
-    await page.route(
-      "**/v1/projects/rad:git:hnrkdi8be7n4hhqoz9rpzrgd68u9dr3zsxgmy/tree/fcc929424b82984b7cbff9c01d2e20d9b1249842/markdown/",
-      route => route.fulfill({ status: 500 }),
-    );
-
-    await page.goto(projectFixtureUrl);
-
-    const sourceTree = page.locator(".source-tree");
-    await sourceTree.locator("text=markdown/").click();
-
-    await expect(
-      page.locator("text=Not able to expand directory"),
-    ).toBeVisible();
-  });
-  test("error appears when file can't be loaded", async ({ page }) => {
-    await page.route(
-      "**/v1/projects/rad:git:hnrkdi8be7n4hhqoz9rpzrgd68u9dr3zsxgmy/blob/fcc929424b82984b7cbff9c01d2e20d9b1249842/.hidden",
-      route => route.fulfill({ status: 500 }),
-    );
-
-    await page.goto(projectFixtureUrl);
-    await page.locator("text=.hidden").click();
-
-    await expect(page.locator("text=Not able to load file")).toBeVisible();
-  });
-  test("error appears when README can't be loaded", async ({ page }) => {
-    await page.route(
-      "**/v1/projects/rad:git:hnrkdi8be7n4hhqoz9rpzrgd68u9dr3zsxgmy/readme/fcc929424b82984b7cbff9c01d2e20d9b1249842",
-      route => route.fulfill({ status: 500 }),
-    );
-
-    await page.goto(projectFixtureUrl);
-    await expect(
-      page.locator("text=The README could not be loaded."),
-    ).toBeVisible();
-  });
-  test("error appears when navigating to missing file", async ({ page }) => {
-    await page.route(
-      "**/v1/projects/rad:git:hnrkdi8be7n4hhqoz9rpzrgd68u9dr3zsxgmy/blob/fcc929424b82984b7cbff9c01d2e20d9b1249842/.hidden",
-      route => route.fulfill({ status: 500 }),
-    );
-
-    await page.goto(`${projectFixtureUrl}/tree/master/.hidden`);
-
-    await expect(page.locator("text=Not able to load file")).toBeVisible();
-  });
-  test("error appears when a image with a relative path can't be loaded", async ({
-    page,
-  }) => {
-    await page.route(
-      "**/v1/projects/rad:git:hnrkdi8be7n4hhqoz9rpzrgd68u9dr3zsxgmy/blob/fcc929424b82984b7cbff9c01d2e20d9b1249842/src/black-square.png",
-      route => route.fulfill({ status: 404 }),
-    );
-
-    await page.goto(projectFixtureUrl);
-    const sourceTree = page.locator(".source-tree");
-    await sourceTree.locator("text=markdown/").click();
-    await sourceTree.locator("text=loading-image.md").click();
-
-    // By having a relative path, this gives away that the image has not loaded
-    // else it would have been converted into a data base64 string
-    await expect(
-      page.locator("img[src='../src/black-square.png']"),
-    ).toBeVisible();
-  });
-});
+*/
+
+//test.describe("browser error handling", () => {
```

Comment:
Can we use either [test.skip()](https://playwright.dev/docs/api/class-test#test-skip-1) or [test.fixme()](https://playwright.dev/docs/api/class-test#test-fixme-1) for tests that aren't working yet?

Reply:
[here](https://github.com/radicle-dev/radicle-interface/pull/515/commits/1483e5fc7093f4ca95b94548181260254d94ced6)